spec/adaptors/quickbooks_online/operations/ledger_class/update_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+support :adaptor_helpers
+
+RSpec.describe LedgerSync::Adaptors::QuickBooksOnline::LedgerClass::Operations::Update do
+  include AdaptorHelpers
+
+  let(:ledger_class) { LedgerSync::LedgerClass.new(ledger_id: '123', name: 'Test', active: true, sub_class: false)}
+
+  it do
```

Comment:
We don't need to do this in this PR, but we could DRY up a lot of these specs using rspec shared examples, given that not much changes here.